### PR TITLE
feat: supplier verification flow

### PR DESCRIPTION
**Pull request by renanpontez**

### CODE REVIEW

**Comment by femacedogomes** [comment]
Everything looks great for me. Just need to resolve merge conflicts.

File: src/app/admin/parceiros/page.tsx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+  AccordionInfo,
+  ButtonSecondary,
+  LoadingComponent,
+  SupplierCard,
+} from "@/components";
+import SupplierLogo from "@/images/easytolive/logo/logotipo-fundoazulroxo.svg";
+import SearchCategory from "@/app/searchCategory";
+import getDateFormater from "@/utils/getDateFormater";
+
+interface Supplier {
```
Comment (pedrohbl03) [inline src/app/admin/parceiros/page.tsx:18]: ISupplier

File: src/components/atoms/AccordionInfo/index.tsx [hunk 2]
```diff
@@ -1,21 +1,27 @@
...
 import { Disclosure } from "@/app/headlessui";
 import { MinusIcon, PlusIcon } from "@heroicons/react/24/outline";
 import React, { FC } from "react";
 
+type DataProps = {
+  name: string | ReactNode;
+  content: string | ReactNode;
+};
+
+interface Props {
```
Comment (pedrohbl03) [inline src/components/atoms/AccordionInfo/index.tsx:13]: IAccordionInfoProps

File: src/components/mols/SupplierCard/index.tsx [hunk 3]
```diff
@@ -7,12 +7,14 @@ import ArrowRight from "@/images/easytolive/icons/arrow-next-right-black.svg";
 import { useRouter } from "next/navigation";
 
 interface SupplierCardProps {
```
Comment (pedrohbl03) [inline src/components/mols/SupplierCard/index.tsx:9]: ISupplierCardProps

**Review by pedrohbl03 — APPROVED** [review]

File: src/app/admin/parceiros/page.tsx (hunk 1 above)
Comment (renanpontez) [inline src/app/admin/parceiros/page.tsx:18]:
```suggestion
interface ISupplier {

```

File: src/app/admin/parceiros/page.tsx [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+  id: string;
+  name: string;
+  email: string;
+  isVerified: boolean;
+  createdAt: string;
+  supplierInfo: any;
+}
+
+function SupplierPage() {
+  const [suppliers, setSuppliers] = useState<Supplier[]>([]);
```
Comment (renanpontez) [inline src/app/admin/parceiros/page.tsx:29]:
```suggestion
  const [suppliers, setSuppliers] = useState<ISupplier[]>([]);

```

File: src/app/admin/parceiros/page.tsx (hunk 1 above)
Comment (renanpontez) [inline src/app/admin/parceiros/page.tsx]:
```suggestion
interface ISupplier {

```

**Comment by pedrohbl03** [comment]
I can merge this? 

Conflicts are removed and everthing sound's good for me.

File: src/app/(application)/admin/parceiros/page.tsx [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+import { useDebounce } from "use-debounce";
+import supplierService from "@/service/supplier.service";
+import { ISupplierList } from "@/types/supplier";
+import { showToastify } from "@/hooks/showToastify";
+import {
+  AccordionInfo,
+  ButtonSecondary,
+  LoadingComponent,
+  SearchCategory,
+  // ButtonThird,
```
Comment (pedrohbl03) [inline src/app/(application)/admin/parceiros/page.tsx:13]:
```suggestion
```

**Review by pedrohbl03 — APPROVED** [review]
You are the guy! 

Sound's good.

**Pull request merged**